lua/pathogen/init.lua

```diff
@@ -312,7 +313,8 @@ end
 
 local function start_builtin(opts)
     opts = opts or {}
-    opts.cwd = opts.cwd or vim.loop.cwd()
+    local cwd = opts.cwd or vim.loop.cwd()
+    opts.cwd = M.short_prompt_path and vim.fnamemodify(cwd, ":~") or cwd
```

Comment:
Does this work for Windows?

Reply:
I don't have a windows machine where I can test this but trying `:lua print(vim.fn.fnamemodify("C:\\test\\test", ":~"))` outputs `~/C:\test\test` so I would say no, unfortunately

Would you prefer an OS check or not implementing this change at all?

Reply:
It should not break on Windows at least, according to your test it would probably do.